app/v1/models/errors/mtdErrors.scala

```diff
@@ -38,34 +44,41 @@ object TypeOfBusinessFormatError
 object AdjustedStatusFormatError
   extends MtdError("FORMAT_ADJUSTED_STATUS", "The supplied adjusted status format is invalid")
 
+object FormatAdjustmentValueError extends MtdError("FORMAT_ADJUSTMENT_VALUE", "") {
+  def withFieldName(fieldName: String): MtdError = this.copy(message = s"The format of the '$fieldName' value is invalid")
```

Comment:
This doesn't match the message in the requirement spec "The format of [fieldName] value is invalid"

Reply:
It does now.